### DISCUSSION TITLE
Is there a way to hide OJS Runtime Errors that appear when first loading an HTML page?

### DISCUSSION THREAD

**Discussion by thegilpatrick** [quarto-dev/quarto-cli#6276] [Q&A]
### Description

Hey I have an imported function that takes a second to load on my webpage. [While it is loading it displays a OJS runtime error](https://thegilpatrick.com/posts/pet-cats-uk/). Is there anyway to hide this error message?

I tried this within the code cell but it didn't work.
```{ojs}
//| warning: false
//| error: false
```

**Comment by cscheid** [comment]
This is indeed a super annoying thing about our OJS support.

I would love to fix it, but I haven't been able to figure out how to detect and wait for the dependency injection from the import statements that happen asynchronously.

**Comment by boshek** [comment]
@thegilpatrick question: Is this in reference to an ojs chunk that truly did fail _or_ is it the situation when a page takes a bit to load and one temporarily sees an ojs chunk failure and then the code executes and it disappears?

**Reply by thegilpatrick** [reply]
Yeah it is that it takes too long to load and displays as an error but then it disappears after. But there is no real error.

**Reply by boshek** [reply]
thanks for the response. Encountering the same issue :(

**Reply by mikabr** [reply]
I'm having the same issue – [here's another example](https://langcog.github.io/numberbank/data.html) for reference.

![numberbank_error](https://github.com/quarto-dev/quarto-cli/assets/1414971/9c71393b-958f-4a78-a245-202eab187082)

**Reply by cscheid** [reply]
Thanks, folks. To be clear, we are keenly aware that the issue exists. It's just not something we believe we can easily fix at the moment.

**Reply by boshek** [reply]
Thanks @cscheid, totally. I just wanted to make sure that this was same issue I was having (which it was). Thanks for all you do!

**Comment by thegilpatrick** [comment]
Hi @cscheid are there any updates on this? Should I submit it as an issue? Or is it still not an easily fixable issue? 

Thanks again for your work on this project!

**Comment by mikabr** [comment]
FWIW a partial workaround I've used is to hide the errors using css:
```
.observablehq--error {
  display: none;
}
```

**Reply by thegilpatrick** [reply]
Oh thanks! Idk why I didn't think of this.
